functions/index.js

```diff
@@ -247,6 +247,18 @@ function createRoomAndJoin(league, roomsList, username, id) {
   return roomID.toString();
 }
 
+function updateUserStats(starIncrease,trophieIncrease,uid) {
```

Comment:
Where is this function called. If it needs to be called by android client you need to export it.

Reply:
What do you mean?

Reply:
It's already called

Reply:
Here you declare your function but I do not see where it is called in this file (index.js)

Reply:
Ohh you are right, I don't use it right now. I just created it incase we want to use it later for the server side statistics update.